open-api/rest-catalog-open-api.yaml

```diff
@@ -1630,6 +1654,98 @@ components:
         - $ref: '#/components/schemas/OAuthClientCredentialsRequest'
         - $ref: '#/components/schemas/OAuthTokenExchangeRequest'
 
+    ScanReportRequest:
+      type: object
+      required:
+        - scan-report
+      properties:
+        scan-report:
+          $ref: '#/components/schemas/ScanReport'
+      example:
+        {
```

Comment:
Is there much value to having a big example like this? I think in most other cases, we leave it out or try to have examples further down the tree structure.

Reply:
I mainly wanted to show an example of the metrics. I removed this big example and added a metrics example directly to `    ScanReport`